package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "todo_list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack --mode development --watch",
```

Comment:
- [ ] As requested by the previous code reviewer, please kindly add a script for running your app here like for example: `"start": "webpack serve --open"`, now one can run your app with `npm start` command. Please kindly fix that. Make sure you replace line 7 with a different script command.